apps/batch-script/batch-script-buildpack/bin/build.bat

```diff
@@ -8,6 +8,7 @@ set layers_dir=%1
 echo [[processes]]       >> %layers_dir%\launch.toml
 echo type = "web"        >> %layers_dir%\launch.toml
 echo command = ["app.bat"] >> %layers_dir%\launch.toml
+echo default = "true" >> %layers_dir%\launch.toml
```

Comment:
Should the `true` be quoted or unquoted here?

Reply:
Oops sorry, It should be unquoted here, because, In TOML syntax, boolean values are represented without quotes to distinguish them from string values...

Reply:
@AidanDelaney ,I have corrected this typo.. 
Actually I corrected the format for Linux previously but forgot to update it for Windows.